### PR TITLE
enable slenkins tests

### PR DESCRIPTION
**Pull request by nadvornik**
depends on
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/527

### CODE REVIEW

**Comment by coolo** [comment]
can you move that into a function, so it's a bit more readable?

**Comment by nadvornik** [comment]
done

File: main.pm [hunk 1]
```diff
@@ -626,6 +626,21 @@ sub load_autoyast_tests(){
...
+    if (get_var("SLENKINS_NODEFILE")) {
+        my $node = get_var("SLENKINS_NODE");
+        if ($node && $node ne 'control' ) {
+            loadtest "slenkins/slenkins_node.pm";
+        }
+        else {
+            loadtest "slenkins/slenkins_control.pm";
+        }
+        $anyloaded = 1;
+    }
```
Comment (coolo) [inline main.pm]: aehm, what is the purpose of this variable. Just a return 1 here and no return otherwise would be clearer IMO

**Comment by nadvornik** [comment]
I copied this from another function

**Pull request merged**